src/themes/masschallenge/modules/modal.overrides

```diff
@@ -17,33 +17,96 @@
 }
 
 .ui.modal > .teal.segment {
-  background-color:  #b8dde4 !important;
+  background-color: #b8dde4 !important;
 }
 
 .ui.modal.rbcCalendar {
-  margin-top: 4rem; 
-  max-width: 368px; 
+  margin-top: 4rem;
+  max-width: 368px;
   min-width: 320px;
 }
 
 .ui.modal.rbcCalendar .scrolling.content {
-    max-height: calc(85vh - 326px);
+  max-height: calc(85vh - 326px);
 }
 
 .ui.modal > .header > .close {
-  position: absolute; 
+  position: absolute;
   right: 10px;
 }
+
 .ui.mini.round.button {
   border-radius: 50%;
   padding: .75em;
   height: 28px;
   width: 28px;
 }
+
 .ui.mini.default.round.button:not(:hover) {
   background-color: initial;
 }
 
 .ui.modal > .header .rbc-calendar {
-    margin: 0 -.5em;
-}
\ No newline at end of file
+  margin: 0 -.5em;
+}
+
+.ui.tiny.modal.cancel-reservation-modal {
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: 600;
+  width: 450px;
+}
+
+.ui.header .content.modal-header {
+  font-weight: bold;
```

Comment:
Elsewhere font-weight is specified numerically which tends to be more accurate. Any reason for using `bold` here instead?